frontend/src/components/new-sightings-page/NewSightingForm.tsx

```diff
@@ -67,6 +67,23 @@ export const NewSightingForm: React.FC<NewSightingFormProps> = ({
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setLocationInputType(event.target.value as LocationInputType);
+    setFormValues({
+      ...formValues,
+      longitude: "",
+      latitude: "",
+    });
+  };
+
+  const getLocationFromPhone = () => {
+    function success(pos: GeolocationPosition) {
+      const crd = pos.coords;
+      setFormValues({
+        ...formValues,
+        latitude: crd.latitude.toString(),
+        longitude: crd.longitude.toString(),
```

Comment:
You can destructure this:
```suggestion
      const { latitude, longitude } = pos.coords;
      setFormValues({
        ...formValues,
        latitude: latitude.toString(),
        longitude: longitude.toString(),
```